src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = 40 * days - 20;
+
+    return sum;
+  }
+
+  if (days >= 7) {
+    sum = 40 * days - 50;
+
+    return sum;
```

Comment:
As with the previous condition, you can return the result directly without using the `sum` variable, which simplifies the code as recommended by checklist item 4.